### ISSUE TITLE
Is anyone maintaining this repo? If not, has everyone moved to a newer better alternative?

### ISSUE THREAD

**Issue by mannharleen** [blaze/odo#619]

**Comment by predicador37** [comment]
I agree with the question. Odo last release was at Apr'16. This release does not work with Pandas 0.24, for example, because of the use of pandas tslib attribute. Although this is fixed in the repo, there's no package with the fix in pypi. Are the maintainers planning any release soon?
Thanks in advance

**Comment by timcera** [comment]
I am concerned also.  What's going on?

**Comment by timcera** [comment]
Should I just take my own version of odo local to my project?  This is painful.  Especially because all that needs to happen is someone to have a new release.  I think that all blockers to my project are fixed in the repo - mainly tslib is not in pandas any more - https://travis-ci.org/timcera/tsgettoolbox

**Comment by branning** [comment]
See #600, it seems there was kind of a plan to migrate to another organization, but that hasn't happened yet.

**Comment by mujina93** [comment]
Any news on this?
